cfme/tests/containers/test_topology.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui.topology import Topology
+from cfme.containers.topology import ContainerTopology
+from cfme.fixtures.pytest_selenium import is_displayed_text
+from random import choice
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP-9996
+
+
+def test_topology_display_names():
+    '''Testing Display Names functionality in Topology view
+    Step:
+        In Topology screen click on the Display names check box
+    Expected result:
+            The entities names should toggle on/off
+            according to the state of the checkbox.'''
+    topo_obj = Topology(ContainerTopology)
+
+    topo_obj.display_names.enable(True)
```

Comment:
consider paramaterizing on True/False and asserting is_displayed_text against the parameter. Reduces the code duplication here.